### ISSUE TITLE
Bad usability of website

### ISSUE THREAD

**Issue by UrtsiSantsi** [gircore/gir.core#939]
![image](https://github.com/gircore/gir.core/assets/142679804/f8a71ab7-f4bd-4b3c-a71c-c4046c6bf836)

The website has usability problems: 

1. "Api Documentation" looks insensitive / disabled and it is a single item - it took me several visits to the website and accidental click on it to realize that it leads to the API documentation, I was under the impression that it is just missing. It also has a very small font - even "Use" and "Contribute" are bigger, let alone the huge "Welcome to Gir.Core".
2. "Home" should be another item before "Api Documentation" instead of relying on the icon, to make it clear that there are 2 main sections and resolve the first problem.
3. A very small font of the main text.
4. A lot of empty space on the left and even more on the right on the "Home page".
5. The indentation on the right side between child and parent is too small - "Fields" and "FontPropertyDefinition" have indentation of just a few pixels, less than one capital letter: 
![image](https://github.com/gircore/gir.core/assets/142679804/cd9bc299-9ae1-4542-bda9-85c2c48654f5)

This problems can be solved with better template like [DocFX Material](https://ovasquez.github.io/docfx-material/) or [SingulinkFX](https://github.com/Singulink/SingulinkFX) and adding the "Home page" as a menu item.

The information can be better:
1. What does this mean "Currently there are multiple libraries available to get started with C# development for Linux: "? For me it reads like I can start working on this libraries, not using them. How about "Currently there are bindings for multiple libraries available to use with C# on Linux: "?
2. The screenshot for GTK looks quite unprofessional and won't make anybody pick it for development. A better alternative is to use GTK Factory or some complex application, like Inkscape or Gimp.
3. A screenshot of one or more random Gnome Circle apps for Adwaita - people love screenshots.
4. The screenshot of GStreamer is also quite basic. How about some video player, like Celluloid?
5. For WebKitGTK there can be a screenshot of Epiphany.

**Comment by fsobolev** [comment]
> The screenshot of GStreamer is also quite basic. How about some video player, like Celluloid?

Celluloid uses libmpv, not GStreamer. Clapper is probably the most impressive player that uses GStreamer and GTK.

**Comment by badcel** [comment]
The templates look nice. If anyone is interested in trying it out locally and providing feedback here is very welcome.

i agree that the introduction text of the libraries page is not optimal. Your suggestion sounds a lot better to me. Feel free to open a PR. The file is located here: https://github.com/gircore/gir.core/blob/main/docs/docs/libraries.md

Regarding the images those are screenshots of the samples (the GTK one is still GTK 3 and the sample got replaced by simpler ones).

I think the GStreamer image is fine as it should not illustrate that GStreamer is a media player but a framework to play media streams.

On the other hand I agree that the GTK one is pretty bad and should be replaced. I think another screenshot from the samples e.g. the about dialog should be sufficient.

The libraries section is not the place to show great apps but to illustrate the core of a library in a distilled form.

To show off great apps there is the [Apps](https://gircore.github.io/docs/apps.html) page which feature apps written with C#. I think using an image of e.g. Gimp would imply that gimp is somehow using gir.core which is not the case.

**Comment by UrtsiSantsi** [comment]
> The libraries section is not the place to show great apps but to illustrate the core of a library in a distilled form.

You are wrong @badcel, it is place to show possible developers what a great things can be done with these libraries. And what better way to show it than impressive looking apps using these libraries, even if they are not using gir.core.
The apps section is a place to show impressive apps that use gir.core.
